Display autocomplete directions

emmykg38 opened the pull request:
Here's what we did:

- utilized the autocomplete data to generate a directions renderer
- removed th:elements in the index file to prevent errors from model binding (all th statements are gone!) -- validation code is still happening inside the trip class and will need to be addressed at some point
- added a map to the index.html file to immediately render directions (needs styling/wireframing and design review)

mgunnhawkins commented on src/main/resources/templates/index.html:
```diff
@@ -2,52 +2,43 @@
+
+
+</div>
+
```
Remove space

mgunnhawkins reviewed (changes requested):
Hi all!  Great work!
I have a few questions about this branch:
1.  I see that you deleted the detour radius to this page.  Where are we planning to add that in?
2. I also noticed that you removed the name your trip portion.  Where are we planning to add that in?
3. I checked my sql database and noticed that the autocomplete queries are not saving as data into MySQL  are they doing that for you?

Thank you!

valerietonsor reviewed (changes requested):
Hi team! Can you please explain the plan moving forward in regards to deleting the th elements?

emmykg38 commented:
This branch is ready to be reviewed again.
More things have changed:

- renamed HomeController to CreateTripController to better describe the functionality
- took out methods in renamed CreateTripController that dealt with model binding of the trip object (which is not possible right now)
- inside AuthenticationController, we redirected to 'planATrip' template to display map and input directions
- (removed displayMap and replaced it with said planATrip template)
- modified the buttons in the header fragment to reflect changes above
- took out marker and window capability and put it inside a separate function, embedded inside initMap()
- index file now displays only the video and a register button (we did not style or add anything else in here, did not feel it was pertinent to our story)
- changed 'Login/Register' in header fragment to 'Login'
- confirmed that we can drive to anchorage but not to alaska, will need to address status error message for directions renderer in another story
- fingers crossed, if this merges well, this FIXES MAIN

Thank you for your thoughtful review!! (:
